openfisca_france_local/departements/eure_et_loir/caracteristique_socio_demographique.py

```diff
@@ -0,0 +1,36 @@
+from openfisca_france.model.base import Variable, Individu, MONTH
+
+class aide_menagere_fournie_caisse_retraite(Variable):
+    value_type = bool
+    label ="Prestations d’aide-ménagère servie par les caisses de retraite"
+    entity = Individu
+    definition_period = MONTH
+    default_value = False
+
+class actp(Variable):
```

Comment:
Indiquer qu'il s'agit de l'éligibilité dans le nom de la variable, par convention ?
```suggestion
class eligibilite_actp(Variable):
```